<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>240</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>420</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>comhdlc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>13</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>12</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonConnect">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>41</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonDisconnect">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>77</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="file_dialog">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="selected_file_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSendFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>261</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_log">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <zorder>selected_file_name</zorder>
   <zorder>groupBox</zorder>
   <zorder>file_dialog</zorder>
   <zorder>buttonSendFile</zorder>
   <zorder>text_log</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
